src/libraries/System.Runtime.InteropServices.RuntimeInformation/ref/System.Runtime.InteropServices.RuntimeInformation.cs

```diff
@@ -21,8 +21,10 @@ public enum Architecture
         private readonly int _dummyPrimitive;
         public static System.Runtime.InteropServices.OSPlatform Browser { get { throw null; } }
         public static System.Runtime.InteropServices.OSPlatform FreeBSD { get { throw null; } }
+        public static System.Runtime.InteropServices.OSPlatform iOS { get { throw null; } }
```

Comment:
instead of adding this to the ref (which will be done in https://github.com/dotnet/runtime/pull/36704 and needs API review), please instead use `RuntimeInformation.IsOSPlatform(OSPlatform.Create("IOS"))` in the usages.

Reply:
Should a follow up after #36704 be to change them back to `RuntimeInformation.IsOSPlatform(OSPlatform.iOS)` and `RuntimeInformation.IsOSPlatform(OSPlatform.tvOS)`?

Reply:
ideally yes, but it's just a cosmetic cleanup